osu.Game/OsuGameBase.cs

```diff
@@ -69,7 +69,7 @@ public partial class OsuGameBase : Framework.Game, ICanAcceptFiles
         /// </summary>
         private const double global_track_volume_adjust = 0.8;
 
-        public bool UseDevelopmentServer { get; }
+        public virtual bool UseDevelopmentServer { get; }
```

Comment:
For legibility how about moving the set into the property as well?

```suggestion
        public virtual bool UseDevelopmentServer => DebugUtils.IsDebugBuild;
```

Note that this requires the ctor code to also be removed.

It's already cached at the `DebugUtils` end so there shouldn't be any performance concern here. Just feels better than having a virtual-get-only property which is written to elsewhere.